docs/book/src/topics/managedcluster.md

```diff
@@ -629,6 +629,38 @@ Some notes about how this works under the hood:
 
 ## Adopting Existing AKS Clusters
 
+### Option 1: Using the experimental ASO-based API
+
+<!-- markdown-link-check-disable-next-line -->
+The [experimental AzureASOManagedControlPlane and related APIs](/topics/aso.html#experimental-aso-api) support
```

Comment:
The link checker doesn't seem to like linking to a dynamically generated domain-local path like this. The link as it appears in the docs preview here seems to work though: https://deploy-preview-4799--kubernetes-sigs-cluster-api-provider-azure.netlify.app/topics/managedcluster#option-1-using-the-experimental-aso-based-api